scipy/stats/tests/test_stats.py

```diff
@@ -5343,6 +5343,7 @@ def test_ttest_ind_scalar():
     assert np.isnan(p)
 
 
+@pytest.mark.filterwarnings("ignore:Arguments...:DeprecationWarning")
```

Comment:
We test for the warning about deprecated `permutations` below.

I suppose there are no tests for deprecated positional use, but I assume the `_deprecate_positional_args` decorator was already tested, so I'm just testing the new part.